docs/_static/nuts-consent-bridge.yaml

```diff
@@ -96,6 +97,23 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/Identifier'
+        initiatingLegalEntity:
+          $ref: '#/components/schemas/Identifier'
+        initiatingNode:
+          type: string
+          description: The X500 name of the node that initiated the transaction (read-only)
+          example: "CN=test,C=NL,L=IJBergen,O=Nuts"
+        requestDateTime:
```

Comment:
consistency in naming (requestDateTime-latestUpdate), this date-time field has its type postfixed (DateTime), latestUpdate hasn't.

Reply:
true, `changeDateTime` instead of latestUpdate?

Reply:
changeDateTime seems fine although it sounds a bit funny, maybe because it isn't so clear whether it indicates something has happened or will happen. I generally like 'last' and/or 'updated' in that request, so that would lead to something like: lastChangeDateTime/updatedDateTime/lastUpdateDateTime

Reply:
no but we can't change the other one due to backwards compatibility. Also it's a string containing a dateTime. Formats from OAS are not handled correctly by different generators

Reply:
We can switch to `createdAt` and `updatedAt`?